#!/usr/bin/env python
PACKAGE = "robot"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("P_koef", double_t, 0, "P koef for vel error", 0.5, 0, 5)
gen.add("I_koef", double_t, 0, "I koef for vel error", 0.1, 0, 5)
gen.add("D_koef", double_t, 0, "D koef for vel error", 0.1, 0, 5)
gen.add("sf", double_t, 0, "Scaling factor for velocities", 100, 100, 500)



exit(gen.generate(PACKAGE, "robot_node", "Reconfigure"))